sdk/eventhubs/azure-messaging-eventhubs/src/test/java/com/azure/messaging/eventhubs/EventHubProducerAsyncClientTest.java

```diff
@@ -129,7 +129,7 @@ class EventHubProducerAsyncClientTest {
     private EventHubConnectionProcessor connectionProcessor;
     private TracerProvider tracerProvider;
     private ConnectionOptions connectionOptions;
-    private final Scheduler testScheduler = Schedulers.newElastic("test");
+    private final Scheduler testScheduler = Schedulers.newBoundedElastic(10, 10, "test");
```

Comment:
@conniey @anuchandy I basically just threw numbers in here to comply with the API, mind checking if this needs to be anything specific.